### PR TITLE
update for dev mode

### PR DESCRIPTION
**Pull request by gjmwoods**

### CODE REVIEW

File: operations-guide/setting-up-axon-server/command-line.md [hunk 1]
```diff
@@ -8,6 +8,7 @@ For the Axon Server Standard edition the only supported commands are:
 - users
 - register-user
 - delete-user
+- delete-all-events (only in [Development Mode](../../operations-guide/setting-up-axon-server/development-mode.md))
```
Comment (abuijze) [inline operations-guide/setting-up-axon-server/command-line.md]: Just a thought: if we call this operation "purge-events" (or just "purge"), we could also use this for the clustered mode to indicate that we want to delete the events of a removed context.

File: operations-guide/setting-up-axon-server/development-mode.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# Development Mode
+
+You can start Axon Server in development mode which enables some features for development convenience.
+
+These features can be enabled by configuring the following property:
+
+```text
+axoniq.axonserver.devtools.enabled=true
```
Comment (abuijze) [inline operations-guide/setting-up-axon-server/development-mode.md]: suggestion: `axoniq.axonserver.devmode.enabled=true` (as the description calls it development mode)

File: operations-guide/setting-up-axon-server/command-line.md (hunk 1 above)
Comment (idugalic) [reply operations-guide/setting-up-axon-server/command-line.md]: I have another question: Will this only remove events from Axon Server Store? If yes, then what about tracking tokens that exist in our DBs, should we consider this as well, and write about it?

Comment (gjmwoods) [reply operations-guide/setting-up-axon-server/command-line.md]: Yes only the events are removed. I will add a note on that point to the docs

**Pull request merged**